TagsterWebAPI/src/Tagster.Application/Exceptions/ProfileNotFoundException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Net;
+using Tagster.Exception.Models;
+
+namespace Tagster.Application.Exceptions;
+
+public class ProfileNotFoundException : AppException
+{
+    public override string Code => "profile_not_found";
```

Comment:
If I remember we add '_exception' on the end

Reply:
nope, currently all codes are without _exception suffix